go2rtc_client/rest.py

```diff
@@ -11,14 +11,15 @@
 from mashumaro.codecs.basic import BasicDecoder
 from mashumaro.mixins.dict import DataClassDictMixin
 
-from .models import Stream, WebRTCSdpAnswer, WebRTCSdpOffer
+from .models import ApplicationInfo, Stream, WebRTCSdpAnswer, WebRTCSdpOffer
 
 if TYPE_CHECKING:
     from collections.abc import Mapping
 
 _LOGGER = logging.getLogger(__name__)
 
 _API_PREFIX = "/api"
+_SUPPORTED_VERSION: Final = "1.9.4"
```

Comment:
Set to current release version for now
This can be expanded as needed, for example with a version range validated with `AwesomeVersion`